styles.css

```diff
@@ -0,0 +1,165 @@
+@import "https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,900;1,300&display=swap";
+
+* {
+  margin: 0;
+  padding: 0;
+  border: none;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  background-color: #1f2937;
+  color: white;
+  width: 100%;
+}
+
+h3 {
+  margin-left: 10%;
+  display: flex;
+  align-self: center;
+}
+
+#header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header-links {
+  margin-right: 150px;
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+  padding: 0;
+  gap: 25px;
+}
+
+a {
+  color: rgb(211 204 204);
+  text-decoration: none;
+}
+
+#header2 {
+  display: flex;
+  justify-content: space-around;
+}
+
+.header2-left {
+  display: flex;
+  flex-direction: column;
+  width: 30%;
+  padding-left: 60px;
+}
+
+h1 {
+  font-size: 48px;
+  margin-bottom: 0;
+  color: #f9faf8;
+}
+
+li {
+  font-size: 18px;
+  color: #e5e7eb;
+}
+
+.image-placeholder {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: rgb(102 114 124);
+  width: 500px;
+  height: 250px;
+  margin: 70px 85px 100px;
+}
+
+button {
```

Comment:
And concerning the hover effect of the button, although it wasn't among the style to be used for the project, it is recommended that you add it.
as it was used in [this](https://deploy-preview-1--dema-top-landing-page.netlify.app/)